.ci/docker/ubuntu-riscv/Dockerfile

```diff
@@ -0,0 +1,154 @@
+ARG UBUNTU_VERSION
```

Comment:
Please add a comment that this is a cross-compiled docker container and may be re-name folder to `ubuntu-cross-riscv` or something like that